app.py

```diff
@@ -29,6 +29,8 @@
 
 import re
 
+from invesalius.net.remote_control import RemoteControl
```

Comment:
Since this is optional it's better to move this import to the the part you use RemoteControl

Reply:
Done. Amended this change to the latest commit to keep the number of commits a bit smaller.